message_broker_producer.install

```diff
@@ -68,8 +68,7 @@ function message_broker_producer_requirements($phase) {
 
   if ($phase == 'runtime') {
 
-    $phplib_library = libraries_detect('messagebroker-phplib');
-    $config_library = libraries_detect('messagebroker-config');
```

Comment:
Once this is merged I'll add the test for $config_library back in a functional state. Thanks for bring this to my attention.

As is :+1: